### DISCUSSION TITLE
Help adding navigation links to top of every page.

### DISCUSSION THREAD

**Discussion by primalxaxa** [vichan-devel/vichan#778] [Q&A] [answered]
I would appreciate if someone smarter than me could help me out. I am trying to add the Ukko navigation from /templates/boardlist.html to all the other pages. I have added the code to each index.html file which works however, every time someone posts on the board it updates the index.html file and deletes the code I added for the boardlist. 

The code is here: `<a href="https://librechan.org">Home</a> <a href="https://librechan.org/recent.html">Recent</a> <a href="https://librechan.org/ukko/">Everything</a> <a href="https://librechan.org/a/">/a/</a> <a href="https://librechan.org/b/">/b/</a> <a href="https://librechan.org/mu/">/mu/</a> <a href="https://librechan.org/o/">/o/</a> <a href="https://librechan.org/pol/">/pol/</a> <a href="https://librechan.org/r/">/r/</a> <a href="https://librechan.org/r9k/">/r9k/</a> <a href="https://librechan.org/sp/">/sp/</a> <a href="https://librechan.org/t/">/t/</a> <a href="https://librechan.org/tv/">/tv/</a> <a href="https://librechan.org/v/">/v/</a>`

Here is a screenshot of what the code does.
![Screenshot_20240723_090229](https://github.com/user-attachments/assets/9865ecb6-6e87-4974-9148-b8f0e991a9a5)

I am assuming there is a central file to add this to instead of manually adding it to each boards index.html which I now know won't save once the board is updated.
Any help would be much appreciated!

**Comment by perdedora** [comment, marked as answer]
I don't think we actually use boardlist.html.
The boards are inserted in the template by `boardlist` argument passed to `Element`.
Which pulls information from `$config['boards']` and build html.

**Reply by primalxaxa** [reply]
thanks so much for the reply. I was able to get it working!
